FSharpBuild.Directory.Build.targets

```diff
@@ -33,6 +33,54 @@
     <Copy SourceFiles="@(IntermediateFilesToCopy)" DestinationFolder="$(FinalIntermediateOutputPath)\%(RecursiveDir)" />
   </Target>
 
+
+  <ItemDefinitionGroup>
+    <PackageReference>
+      <IsVsDependency>false</IsVsDependency>
+    </PackageReference>
+  </ItemDefinitionGroup>
+
+  <Target Name="PackageFSharpDesignTimeTools" BeforeTargets="CoreCompile" DependsOnTargets="CollectPackageReferences">
+      <ItemGroup>
+        <EnhancedPackageReference Include="@(PackageReference)" Condition = " '%(PackageReference.IsFSharpDesignTimeProvider)' != 'false' and '%(PackageReference.IsFSharpDesignTimeProvider)' != '' ">
+           <PropertyName Condition=" '%(PackageReference.Extension)' != '' ">%(PackageReference.FileName)%(PackageReference.Extension)</PropertyName>
+           <PropertyName Condition=" '%(PackageReference.Extension)' == '' ">%(PackageReference.FileName)</PropertyName>
+        </EnhancedPackageReference>
+
+        <PropertyNames Include = "Pkg$([System.String]::Copy('%(EnhancedPackageReference.PropertyName)').Replace('.','_'))" />
+        <FscCompilerTools Include = "$(%(PropertyNames.Identity))" />
+      </ItemGroup>
+  </Target>
+
+  <!--
+    ============================================================
+                                        MoveVSDependencies
+
+    Copy references that are marked as "CopyLocal" and their dependencies, including .pdbs, .xmls and satellites.
+    ============================================================
+    -->
+  <Target Name="MoveVSDependencies" BeforeTargets = "_CopyFilesMarkedCopyLocal" Condition="'@(ReferenceCopyLocalPaths)' != ''">
```

Comment:
>_CopyFilesMarkedCopyLocal [](start = 53, length = 25)

It is generally not a good idea to depend on private targets defined in msbuild or SDK (underscore-prefixed) as they might change in future. OK for now, but would be good to follow up.